hubverse_annotator/app.py

```diff
@@ -161,8 +217,32 @@ def main() -> None:
         if smhubt_to_plot.is_empty():
             st.warning("No forecasts available for current selection.")
         else:
-            forecast_chart = create_quantile_forecast_chart(smhubt_to_plot)
-            st.altair_chart(forecast_chart, use_container_width=True)
+            forecast_layers = create_quantile_forecast_chart(smhubt_to_plot)
+            if eh_table is not None:
+                eh_to_plot = eh_table.with_columns(pl.col("date")).filter(
+                    (pl.col("location") == two_num_loc_abbr)
+                    & (pl.col("target") == selected_target)
```

Comment:
Can have individual boolean expressions as arguments to `.filter()` and it will parse as "and" (similar to `dplyr::filter()` in R)
```suggestion
                    pl.col("location") == two_num_loc_abbr,
                    pl.col("target") == selected_target
```